package.json

```diff
@@ -20,9 +21,10 @@
     "watch": "npm run start",
     "develop": "npm run start",
     "build": "lerna run build --stream --no-private --ignore @builder.io/plugin-loader --ignore @builder.io/widgets --ignore @builder.io/plugin-shopify",
-    "release:patch": "npm run verify-git-not-dirty && npm run build && lerna version patch --yes && lerna run build:unpkg:dist --no-private && lerna publish from-package --yes",
-    "release:minor": "npm run verify-git-not-dirty && npm run build && lerna version minor --yes && lerna run build:unpkg:dist --no-private && lerna publish from-package --yes",
-    "release:prerelease:dev": "npm run build && lerna version prerelease --yes && lerna run build:unpkg:dist --no-private && lerna publish from-package --dist-tag dev --yes",
-    "release:dev": "npm run release:prerelease:dev"
+    "release:patch": "npm run verify-git-not-dirty && npm run build && lerna version patch --yes && lerna run build:unpkg:dist --no-private && ALLOW_PUBLISH=true lerna publish from-package --yes",
+    "release:minor": "npm run verify-git-not-dirty && npm run build && lerna version minor --yes && lerna run build:unpkg:dist --no-private && ALLOW_PUBLISH=true lerna publish from-package --yes",
+    "release:prerelease:dev": "npm run build && lerna version prerelease --yes && lerna run build:unpkg:dist --no-private && ALLOW_PUBLISH=true lerna publish from-package --dist-tag dev --yes",
```

Comment:
I haven't been able to fully test this one due to memsource error:
```
@builder.io/memsource-connector: > webpack --mode production
@builder.io/memsource-connector: sh: webpack: command not found
```

Reply:
maybe that repo just needs an `npm install webpack --dev`?

also we can have these scripts exclude memsource-connector with a lerna filter

Reply:
did that, should we have lerna ignore all plugins? I removed them from the lerna.json file but can add back and manually ignore this one